src/services/config.js

```diff
@@ -157,9 +169,8 @@ class ConfigData {
     this.prompts = configObj.prompts || {};
     // Note the historical 'projects' naming.
     configObj.projects = configObj.projects || [];
-    configObj.projects.forEach((project) => this.addProfile(project.id, project.accountSid, project.region));
-    this.setActiveProfile(configObj.activeProject);
     this.profiles = configObj.profiles || {};
```

Comment:
Along with loading, you will still need to call addProfile on each entry in the list in case there are any updates to existing ones in the file

Reply:
We shouldn't be calling `addProfiles` on every entry on list now, since with the current change, we'll be copying everything to `profiles` on every config file read in the method. Can you please add on more here for the updates part? I might be missing something here, as In case of updates, we'll still be capturing the list changed via Line #171 ?

Also, couldn't figure out why we needed an extra step to iterate through the list when we could just load the json via `ConfigObj.projects = configObj.projects || []; `. @childish-sambino  Can you please add on more here for this step? If we need, I can add on a method `addProjects` to replicate the existing behaviour for projects.

Reply:
Part of the reason for calling `this.addProfile` was to sanitize the properties. We didn't originally sanitize so some customers created profiles with leading/trailing spaces in the ID and it created problems. We updated it to sanitize when loading the profiles (and when creating profiles) to resolve this.

Also note that just storing in `configObj.projects` doesn't get the projects into `this.projects`.

Reply:
Thanks for the details! 
Updated.